src/main/java/com/thirdcc/webapp/web/rest/UserResource.java

```diff
@@ -177,6 +184,26 @@ public UserResource(UserService userService, UserRepository userRepository, Mail
                 .map(UserDTO::new));
     }
 
+    /**
+     * {@code GET /user/current} : get the current user.
+     *
+     * @return the {@link ResponseEntity} with status {@code 200 (OK)} and with body the userId user, or with status {@code 404 (Not Found)}.
+     */
+    @GetMapping("/users/current")
+    public ResponseEntity<UserDTO> getCurrentLoginUser() {
+        log.debug("REST request to get Current Login User");
+        String userLogin = SecurityUtils
+            .getCurrentUserLogin()
+            .orElseThrow(() -> new BadRequestException("User not Login"));
+        UserDTO userDTO = userService.getUserByLogin(userLogin)
+            .map(UserDTO::new)
+            .orElseThrow(() -> new BadRequestException("User not found"));
+        userCCInfoService.findOneByUserId(userDTO.getId())
+            .ifPresent(userCCInfoDTO -> userService.mapUserWithUserCCInfo(userDTO, userCCInfoDTO));
+        log.debug("User DTO after map: {}", userDTO);
+        return ResponseEntity.ok().body(userDTO);
+    }
+
```

Comment:
@HowardYaw can we do this in GET: /users/:login ? What is the intention of creating this endpoint?

Reply:
we can do in that also
but using this endpoint API consumer no need to pass in any `login`
we use the SecurityUtils to get current/logged in user login